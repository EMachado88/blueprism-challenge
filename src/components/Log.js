const Log = ({ 
  log: {
    serverName,
    status
  }
}) => {
  return (
    <div className='log'>
      <h3>{serverName}</h3>
      <p>{status}</p>
    </div>
  )
}

export default Log