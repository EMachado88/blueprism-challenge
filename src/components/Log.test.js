import renderer from 'react-test-renderer'

import Log from './Log'

test('Renders Log correctly', () => {
  const log = {
    'id': 10375504,
    'startTime': '2021-09-16T20:48:39.953Z',
    'endTime': '2021-06-20T11:32:36.774Z',
    'status': 'Completed',
    'serverName': 'laborum est dolore',
    'scheduleId': 87584078
  }

  const component = renderer.create(
    <Log log={log} />,
  )

  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
