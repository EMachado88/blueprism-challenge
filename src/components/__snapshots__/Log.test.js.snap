// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Log correctly 1`] = `
<div
  className="log"
>
  <h3
    className="text-center"
  >
    laborum est dolore
  </h3>
  <hr />
  <p>
    <strong>
      id:
    </strong>
     
    <span>
      10375504
    </span>
  </p>
  <p>
    <strong>
      Start:
    </strong>
     
    <span>
      9/16/2021
    </span>
  </p>
  <p>
    <strong>
      End:
    </strong>
     
    <span>
      6/20/2021
    </span>
  </p>
  <p>
    <strong>
      Status:
    </strong>
     
    <span>
      Completed
    </span>
  </p>
</div>
`;
